services/QuillLMS/app/helpers/teacher_center_helper.rb

```diff
@@ -46,7 +45,7 @@ def teacher_center_tabs(large: true)
         url: 'faq'
       }
     ]
-    tabs.insert(1, comprehension_tab) if is_comprehension_user
+    tabs.insert(1, comprehension_tab)
```

Comment:
Since there is no longer a conditional here it could just be added to the array directly.